tests/test_browser.py

```diff
@@ -0,0 +1,77 @@
+from os import getenv
+from pathlib import Path
+from shutil import which
+from subprocess import check_output
+from uuid import uuid4
+
+from PIL import Image, ImageChops
+from playwright.sync_api import expect
+
+HERE = Path(__file__).absolute().parent
+
+CONTAINER_ID = getenv("CONTAINER_ID", "test")
+JUPYTER_HOST = getenv("JUPYTER_HOST", "http://localhost:8888")
+JUPYTER_TOKEN = getenv("JUPYTER_TOKEN", "secret")
+
+
+def compare_screenshot(test_image, threshold=2):
```

Comment:
https://playwright.dev/docs/test-snapshots is not available in Python, this is just a very basic image diff.